Sources/SwiftSyntaxBuilderGeneration/gyb_generated/Trivia.swift

```diff
@@ -0,0 +1,108 @@
+//// Automatically Generated From Trivia.swift.gyb.
+//// Do Not Edit Directly!
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+public class Trivia {
+  public let name: String
+  public let comment: String
+  public let serializationCode: Int
+  public let characters: [Character]
+  public let swiftCharacters: [Character]
+  public let isNewLine: Bool
+  public let isComment: Bool
+  
+  public var lowerName: String { lowercaseFirstWord(name: name) }
+  
+  public var charactersLen: Int { characters.count }
+  
+  public var isCollection: Bool { charactersLen > 0 }
+  
+  public init(name: String,
+              comment: String,
+              serializationCode: Int, characters: [Character] = [],
+              swiftCharacters: [Character] = [],
+              isNewLine: Bool = false,
+              isComment: Bool = false){
+    self.name = name
+    self.comment = comment
+    self.serializationCode = serializationCode
+    self.isNewLine = isNewLine
+    self.isComment = isComment
+    
+    
+    // Swift sometimes doesn't support escaped characters like \f or \v;
+    // we should allow specifying alternatives explicitly.
+    if !swiftCharacters.isEmpty {
+      self.swiftCharacters = swiftCharacters
+    } else {
+      self.swiftCharacters = characters
+    }
+  }
+}
+
+
+let TRIVIAS: [Trivia] = [
+  Trivia(name: "Space",
+         comment: "A space ' ' character.",
+         serializationCode: 0),
+  Trivia(name: "Tab",
+         comment: "A tab '\t' character.",
+         serializationCode: 1),
+  Trivia(name: "VerticalTab",
+         comment: "A vertical tab '\v' character.",
```

Comment:
The `\v` produces a 'Unterminated string literal'.
How should we solve this? Is there a way to escape things?

Reply:
We could use raw string literals
```suggestion
         comment: #"A vertical tab '\v' character."#,
```